nservicebus/errors/automatic-retries.md

```diff
@@ -70,6 +70,10 @@ NOTE: Retrying messages for extended periods of time would hide failures from op
 
 SLR can be configured in several ways.
 
+### Transport transaction requirements
+
+The SLR mechanism is implemented by rolling back the [transport transaction](/nservicebus/messaging/transactions.md) and scheduling message for [delayed-delivery](/nservicebus/messaging/delayed-delivery.md) the next time it is picked from the input queue. Aborting the receive operation when transactions are turned off would results in a message loss. Therefore SLR cannot be used when transport transactions are disabled.
```

Comment:
> FLR does the counting semantics correct?

I don't understand, what do you mean?

> It might help here to list the steps to understand the behavior.

You mean first FLR, then SLR, then EQ? Can you give an example of what you had in mind? I'm not sure if going very detailed would be useful for the user.